### ISSUE TITLE
Access Editor: Email address verification overhaul

### ISSUE THREAD

**Issue by b18alegu** [HGustavs/LenaSYS#9133]
When a new user is created or imported to LenaSYS a check of its email is done within the `verifyUserInputForm`-function. This test however is not exhaustive. A separate function should be created to do a more thorough verification of the given email address, and that function should then be called from `verifyUserInputForm`.

For example an email with multiple **@**-marks and addresses containing space(s) will be allowed now. Though further investigation should be done prior to implementation.

Work on this will commence when issue #8935 is solved, verified and merged into master.

**Comment by a18viksa** [comment]
Closing issue after merge to master.
